requirements/manifest.json

```
@@ -11,7 +11,7 @@
         },
         "megatron-lm": {
             "repo": "https://github.com/NVIDIA/Megatron-LM",
-            "ref": "791b70e031dc6905f77e29c25f1e2ac5b8dd4870"
+            "ref": "f62eb5b5c2bc75cb6e0c1d7a7c0813f84b223291"
```

```suggestion
            "ref": "40a13fd985a840042255da504d22d5b2c4e1d173"
```

Let use this one as to make sure to reference the commit with the version bump